src/components/button/Button.tsx

```diff
@@ -50,10 +68,18 @@ export const Button = ({
         `storybook-button--${size} ${className}`,
         mode,
       ].join(' ')}
+      data-testid="storybook-button-test_id"
+      disabled={disable}
       style={{ backgroundColor }}
       {...props}
     >
       {label}
+
+      {icon ? (
```

Comment:
use this `{icon && <img src={arrow} style={{ marginTop: '5px', marginLeft: '5px' }} />}`